sycl/source/detail/stream_impl.cpp

```diff
@@ -61,6 +61,29 @@ size_t stream_impl::get_size() const { return BufferSize_; }
 
 size_t stream_impl::get_max_statement_size() const { return MaxStatementSize_; }
 
+void stream_impl::fill(QueueImplPtr Queue) {
```

Comment:
Could you please clarify if there is any reason why this function is a method of stream_imp?
I suggest moving it somewhere inside the library(scheduler_helpers.hpp/stream_helpers.hpp?) to avoid adding new API.

Reply:
Moved as a function to separate helper header.